### PR TITLE
update cephfs volume

### PR DESCRIPTION
**Pull request by carlory**
k/k: https://github.com/kubernetes/kubernetes/pull/124544 (merged)
preview: https://deploy-preview-46363--kubernetes-io-main-staging.netlify.app/docs/concepts/storage/volumes/#cephfs

### CODE REVIEW

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -534,18 +534,20 @@ please install corresponding CSI drivers.
 
 Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
 
-* `photonPersistentDisk` - Photon controller persistent disk.
-  (**not available** starting v1.15)
-* `scaleIO` - ScaleIO volume.
-  (**not available** starting v1.21)
+* [`cephfs`](/docs/concepts/storage/volumes/#cephfs)
```
Comment (salaxander) [inline content/en/docs/concepts/storage/persistent-volumes.md:537]: is `cephfs` still intended to be listed on line 522? If so, could you also update the removal version there?

Comment (carlory) [reply content/en/docs/concepts/storage/persistent-volumes.md:535]: removed.

**Pull request merged**